tests/test_export.sh

```diff
@@ -0,0 +1,64 @@
+# Run from the root of the project using `bash tests/test_export.sh`
+set -e
+echo "Running test_export.sh"
+
+echo "Exporting the export test environment"
+cd src/cli/project/export/test-data/testenv
+pixi project export conda-environment | tee test-env.yml
+echo "Creating the export test environment with micromamba"
+micromamba create -y -f test-env.yml -n export-test
```

Comment:
Great that you added this test! I think just running a `--dry-run` of most of this would be enough we only need to test if it loads the file correctly.

Reply:
`--dry-run` doesn't appear to test if the pip dependencies are formatted correctly, and instead tries to activate the environment and hand them off to pip.

```sh
❯ bash tests/test_export.sh
Running test_export.sh
Exporting the export test environment
name: default
channels:
- conda-forge
dependencies:
- python >=3.12.5,<4
- pyyaml >=6.0.2,<7
- pip
- pip:
  - rich>=13.8.0, <14

Creating the export test environment with micromamba
conda-forge/osx-arm64                               11.8MB @  14.0MB/s  0.8s
conda-forge/noarch                                  16.4MB @  16.0MB/s  1.0s

Transaction

  Prefix: /Users/akerney/micromamba/envs/export-test

  Updating specs:

   - python[version='>=3.12.5,<4']
   - pyyaml[version='>=6.0.2,<7']
   - pip


  Package              Version  Build               Channel           Size
────────────────────────────────────────────────────────────────────────────
  Install:
────────────────────────────────────────────────────────────────────────────

  + xz                   5.2.6  h57fd34a_0          conda-forge     Cached
  + libexpat             2.6.3  hf9b8971_0          conda-forge     Cached
  + ncurses                6.5  h7bae524_1          conda-forge     Cached
  + yaml                 0.2.5  h3422bc3_2          conda-forge     Cached
  + python_abi            3.12  5_cp312             conda-forge     Cached
  + bzip2                1.0.8  h99b78c6_7          conda-forge     Cached
  + libffi               3.4.2  h3422bc3_5          conda-forge     Cached
  + libzlib              1.3.1  hfb2fe0b_1          conda-forge     Cached
  + ca-certificates  2024.8.30  hf0a4a13_0          conda-forge     Cached
  + readline               8.2  h92ec313_1          conda-forge     Cached
  + tk                  8.6.13  h5083fa2_1          conda-forge     Cached
  + libsqlite           3.46.1  hc14010f_0          conda-forge     Cached
  + openssl              3.3.2  h8359307_0          conda-forge     Cached
  + tzdata               2024a  h8827d51_1          conda-forge     Cached
  + python              3.12.6  h739c21a_0_cpython  conda-forge     Cached
  + wheel               0.44.0  pyhd8ed1ab_0        conda-forge     Cached
  + setuptools          73.0.1  pyhd8ed1ab_0        conda-forge     Cached
  + pip                   24.2  pyh8b19718_1        conda-forge     Cached
  + pyyaml               6.0.2  py312h024a12e_1     conda-forge     Cached

  Summary:

  Install: 19 packages

  Total download: 0 B

────────────────────────────────────────────────────────────────────────────


Dry run. Not executing the transaction.

Installing pip packages: rich>=13.8.0, <14
critical libmamba Cannot activate, prefix does not exist at: '/Users/akerney/micromamba/envs/export-test'
/var/folders/n6/hl07fcc55ys7mbybt01gb8r80000gn/T/mambafD4HVTylLmG: line 5: : command not found
critical libmamba pip failed to install packages
```

Reply:
Oh lol, that sounds like a bug. I'm a bit conflicted here. We're having a pretty big issue with slow CI and this would blow it up more then I would like. Do you think you could only run it if the export code and these tests are touched?

Reply:
We can also make it a test that only runs when a certain label is applied?

Reply:
It looks like it will take an extra job which will check if things have changed, but then export tests could run afterwards. https://stackoverflow.com/a/70711156